src/modules/updatestatus.js

```diff
@@ -0,0 +1,23 @@
+const completedTo = (task) => {
+  task.completed = true;
+};
+
+const incompleteTo = (task) => {
+  task.completed = false;
+};
+
+export { completedTo, incompleteTo };
+
+// export const completedTo = (task) => {
+//   return {
+//     ...task,
+//     completed: true,
+//   };
+// };
+
+// export const incompleteTo = (task) => {
+//   return {
+//     ...task,
+//     completed: false,
+//   };
+// };
```

Comment:
- [OPTIONAL] Kindly remove the commented code in your codebase in order to keep your repo clean and professional.

Reply:
WOW, @levy002 Thank you so much. What a professional and detailed review. i have done this